docs/ReleaseNotes.md

```diff
@@ -18,6 +18,19 @@ The included licenses apply to the following files:
 
 ## Changelog
 
+### Version 1.8.2407
+
+This cumulative release contains numerous bug fixes and stability improvments.
+
+Here are some highlights:
+
+- dxc generates invalid alignment on groupshared matrix load/store instructions in [#6416](https://github.com/microsoft/DirectXShaderCompiler/issues/6416)
+- [Optimization] DXC is missing common factor optimization in some cases in [#6593](https://github.com/microsoft/DirectXShaderCompiler/issues/6593)
+- Fix RemoveUnstructuredLoopExits when an exiting edge jumps out multiple levels of loops in [#6668](https://github.com/microsoft/DirectXShaderCompiler/pull/6668)
```

Comment:
I only just wrote in CONTRIBUTING that we don't allow links to PRs here. This work really should have had a governing bug for it. In conversation with @dneto0, I think he said this isn't worth mentioning in release notes. Unless he objects, I think we should leave this one out.
```suggestion
```